server/src/.mf.config.SAMPLE.json

```diff
@@ -9,18 +9,10 @@
             "name": "core",
             "cliFlag": null,
             "cliShortFlag": null,
-            "fileName": "firebolt-open-rpc.json",
+            "url": "https://meta.rdkcentral.com/firebolt/api/",
```

Comment:
I think the fact that the parent key is "supportedSdks" is problematic now that we have one file that spans multiple SDKs. Maybe this should be something more like "supportedOpenRpcDefinitions" or "supportedInterfaces" or something like that. Or maybe we remove this feature completely somehow. Again, the problem is that there's now no longer a 1-1 mapping between SDKs and OpenRPC files.